tests/smoke-nuget-potential-downgrade.yaml

```diff
@@ -535,32 +500,6 @@ output:
                 - name: xunit.abstractions
                   requirements: []
                   version: 2.0.2
-                - name: Microsoft.VisualStudio.Text.Data
```

Comment:
The reported dependencies have changed because the `Directory.Packages.props` file contains dependencies that aren't listed in the relevant `project.csproj` so the new list is more accurate because only a `.csproj` can contain (or lead to) dependencies.

A `Directory.Packages.props` file could contain a line line this:

``` xml
...
<PackageVersion Include="Some.Unused.Package" Version="1.2.3" />
...
```

The previous version of the NuGet updater (see linked PR) would report this package as a dependency, but it really shouldn't be listed because the starting `.csproj` doesn't contain the corresponding `<PackageReference>` element and if the `Version` attribute was to be updated, no real change would occur in the final package resolution.